win32ss/user/user32/windows/cursoricon.c

```diff
@@ -2651,10 +2926,13 @@ HICON WINAPI CreateIconFromResourceEx(
     if(isAnimated)
         HeapFree(GetProcessHeap(), 0, cursorData.aspcur);
 
+end:
+    HeapFree(GetProcessHeap(), 0, pbBmpIcon);
     return hIcon;
 
     /* Clean up */
 end_error:
```

Comment:
As in the case of `CURSORICON_LoadFromFileW`, it seems it could be possible to merge both the cleanup made above and the end_error cleanup done below.